<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>磁盘臂调度模拟</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="DiskArmSchedulingView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>531</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>调度结果</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>531</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;访问序列(用,隔开 最大磁道编号为500)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="random">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tasks"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>531</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="setting">
       <property name="text">
        <string>设置初始磁头位置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="defaultPos"/>
     </item>
     <item>
      <widget class="QRadioButton" name="SSTF">
       <property name="text">
        <string>SSTF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SCAN">
       <property name="text">
        <string>SCAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="leftDirection">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>←</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rightDirection">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>→</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run">
       <property name="text">
        <string>模拟调度</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>531</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>► 播放动画</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>600</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>电计1503班 何振邦 荣誉出品</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DiskArmSchedulingView</class>
   <extends>QWidget</extends>
   <header>diskarmschedulingview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
